packages/teleterm/src/ui/ClusterConnect/ClusterLogin/useClusterLogin.ts

```diff
@@ -101,6 +102,7 @@ export default function useClusterLogin(props: Props) {
     shouldPromptSsoStatus,
     shouldPromptHardwareKey,
     title: getClusterName(cluster),
+    loggedInUser,
```

Comment:
Its funny you mention that, I originally had it as username and decided to change to `loggedInUser` because I didn't want to confuse the use of it later in the form that had `user`.

I'm happy with `loggedInUserName`